src/fsharp/ConstraintSolver.fsi

```diff
@@ -94,7 +94,7 @@ type ConstraintSolverState =
 
 type ConstraintSolverEnv 
 
-val BakedInTraitConstraintNames : string list
+val BakedInTraitConstraintNames : Set<string>
```

Comment:
Nice but how does it affect performance? I assume insertion might cost more in Set.